app/models/review_assignment.rb

```diff
@@ -162,6 +154,14 @@ def reject_by_no_topic_selection_or_no_submission(contributor_set)
     contributor_set
   end
 
+  def filter_least_reviewed(contributor_set)
```

Comment:
This method `filter_least_reviewed` is exactly the same as `reject_by_least_reviewed`. Why did you rewrite it?